cli/src/test/java/io/specmesh/cli/CliFunctionalTest.java

```diff
@@ -73,7 +73,15 @@ void shouldProvisionTopicsAndAclResourcesWithoutSchemas() throws Exception {
         assertThat(parseResult.matchedArgs().size(), is(4));
 
         // When:
-        kafkaProvisionCommand.run();
+        final var status = kafkaProvisionCommand.call();
+
+        // then: Verify status is correct
+        final var topicProvisionStatus = status.topics();
+        assertThat(topicProvisionStatus.createTopics().size(), is(3));
+        assertThat(topicProvisionStatus.domainTopics().size(), is(3));
+        assertThat(topicProvisionStatus.existingTopics().size(), is(1));
+        final var aclProvisionStatus = status.acls().aclsToCreate();
+        assertThat(aclProvisionStatus.size(), is(12));
```

Comment:
Testing on the weak side again.

Much better to test its actually returning the _right_ data, rather than just the right number of entries.

For example: 
```
assertThat(topicProvisionStatus.createTopics().size(), is(3));
```

Would be better if it was:

```
assertThat(topicProvisionStatus.createTopics(), containsInAnyOrder("blah", "blah2", "other");
```

Reply:
yep - fixed it - thanks